src/main/java/it/gov/pagopa/payhub/activities/connector/pagopapayments/PaymentsReportingPagoPaServiceImpl.java

```diff
@@ -26,4 +26,10 @@ public List<PaymentsReportingIdDTO> getPaymentsReportingList(Long organizationId
 		log.info("Getting payments reporting list for organizationId: {}", organizationId);
 		return paymentsReportingPagoPaClient.getPaymentsReportingList(organizationId, authnService.getAccessToken());
 	}
+
+	@Override
+	public Long fetchPaymentReporting(Long organizationId, String flowId) {
```

Comment:
flowId is ambiguous in our context
```suggestion
	public Long fetchPaymentReporting(Long organizationId, String pagopaPaymentsReportingId) {
```

Reply:
ps: fix also in log message